test/integration/cluster-init/expected/ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -8,6 +8,7 @@ periodics:
     repo: release
   interval: 12h
   labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
```

Comment:
Is this correct?

Reply:
Yeah, I want to verify that that job doesn't disappear when `cluster-init` is run.